Update alert with acknowledge example (#16646)
home-assistant/home-assistant.github.io: source/_integrations/alert.markdown

alert:
  garage_door:

>     name: Garage is open
>     message: The garage door is still open
>     done_message: The garage is closed

Reviewer: ```suggestion
    name: "Garage is open"
    message: "The garage door is still open"
    done_message: "The garage is closed"
```

This particular example relies on the `inline_keyboard` functionality of
Telegram, where the user is presented with buttons to execute certain actions.

Based on the example above you can make an automation to stop further messages,
but you will still receive the done message.

```yaml

> - alias: 'Telegram callback to stop alerts for garage door'

Reviewer: ```suggestion
- alias: "Telegram callback to stop alerts for garage door"
```

>   id: 9823f04d-6229-430b-1fb8-0e959a2385ff

Reviewer: This is optional and should be removed.

```suggestion
```

  trigger:
    - platform: event
      event_type: telegram_callback
      event_data:

>         data: '/garage_acknowledge'

Reviewer: ```suggestion
        data: "/garage_acknowledge"
```

  action:

>     - service: alert.turn_off
>       entity_id: alert.garage_door

Reviewer: ```suggestion
    - service: alert.turn_off
      target:
        entity_id: alert.garage_door
```
